### PR TITLE
Adding base URL for UN URLs

### PR DESCRIPTION
**Pull request by kohsah**
w3id.org placeholder for UN system redirectors (subsequently ontology)

### CODE REVIEW

File: UN/.htaccess [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^$ https://www.un.org/ [R=301,L]
+RewriteRule ^FAO/?$ https://www.fao.org [R=301,L]
```
Comment (davidlehn) [inline UN/.htaccess]: The https link is 404.  Is that a temporary problem or should this be http?

Comment (kohsah) [inline UN/.htaccess]: Oops seems like the www.fao.org is a http site, it should be http then.

Comment (kohsah) [inline UN/.htaccess]: Should I send another PR with the fixed .htaccess ?

Comment (davidlehn) [inline UN/.htaccess]: You can update this one.  Either just commit a correction, or add a commit and squash the two into one commit and force push the result.
